src/openrct2/world/SurfaceData.cpp

```diff
@@ -540,147 +493,147 @@ constexpr uint8_t TileElementLowerStyles[9][32] = {
       0 },
     // MAP_SELECT_TYPE_EDGE_0
-    { SURFACE_STYLE_FLAG_RAISE_OR_LOWER_BASE_HEIGHT | TILE_ELEMENT_SLOPE_N_CORNER_UP | TILE_ELEMENT_SLOPE_E_CORNER_UP,
-      SURFACE_STYLE_FLAG_RAISE_OR_LOWER_BASE_HEIGHT | TILE_ELEMENT_SLOPE_N_CORNER_UP | TILE_ELEMENT_SLOPE_E_CORNER_UP,
-      SURFACE_STYLE_FLAG_RAISE_OR_LOWER_BASE_HEIGHT | TILE_ELEMENT_SLOPE_N_CORNER_UP | TILE_ELEMENT_SLOPE_E_CORNER_UP,
-      SURFACE_STYLE_FLAG_RAISE_OR_LOWER_BASE_HEIGHT | TILE_ELEMENT_SLOPE_N_CORNER_UP | TILE_ELEMENT_SLOPE_E_CORNER_UP,
-      0,
-      TILE_ELEMENT_SLOPE_N_CORNER_UP,
-      TILE_ELEMENT_SLOPE_E_CORNER_UP,
-      TILE_ELEMENT_SLOPE_N_CORNER_UP | TILE_ELEMENT_SLOPE_E_CORNER_UP,
-      0,
-      TILE_ELEMENT_SLOPE_N_CORNER_UP,
-      TILE_ELEMENT_SLOPE_E_CORNER_UP,
-      TILE_ELEMENT_SLOPE_N_CORNER_UP | TILE_ELEMENT_SLOPE_E_CORNER_UP,
-      0,
-      TILE_ELEMENT_SLOPE_N_CORNER_UP,
-      TILE_ELEMENT_SLOPE_E_CORNER_UP,
-      TILE_ELEMENT_SLOPE_N_CORNER_UP | TILE_ELEMENT_SLOPE_E_CORNER_UP,
-      SURFACE_STYLE_FLAG_RAISE_OR_LOWER_BASE_HEIGHT | TILE_ELEMENT_SLOPE_N_CORNER_UP | TILE_ELEMENT_SLOPE_E_CORNER_UP,
-      SURFACE_STYLE_FLAG_RAISE_OR_LOWER_BASE_HEIGHT | TILE_ELEMENT_SLOPE_N_CORNER_UP | TILE_ELEMENT_SLOPE_E_CORNER_UP,
-      SURFACE_STYLE_FLAG_RAISE_OR_LOWER_BASE_HEIGHT | TILE_ELEMENT_SLOPE_N_CORNER_UP | TILE_ELEMENT_SLOPE_E_CORNER_UP,
-      SURFACE_STYLE_FLAG_RAISE_OR_LOWER_BASE_HEIGHT | TILE_ELEMENT_SLOPE_N_CORNER_UP | TILE_ELEMENT_SLOPE_E_CORNER_UP,
-      0,
-      TILE_ELEMENT_SLOPE_N_CORNER_UP,
-      TILE_ELEMENT_SLOPE_E_CORNER_UP,
-      TILE_ELEMENT_SLOPE_N_CORNER_UP | TILE_ELEMENT_SLOPE_E_CORNER_UP,
-      0,
-      TILE_ELEMENT_SLOPE_N_CORNER_UP,
-      TILE_ELEMENT_SLOPE_E_CORNER_UP,
-      TILE_ELEMENT_SLOPE_N_CORNER_UP | TILE_ELEMENT_SLOPE_E_CORNER_UP,
-      0,
-      TILE_ELEMENT_SLOPE_N_CORNER_UP | TILE_ELEMENT_SLOPE_S_CORNER_UP | TILE_ELEMENT_SLOPE_W_CORNER_UP,
-      TILE_ELEMENT_SLOPE_E_CORNER_UP | TILE_ELEMENT_SLOPE_S_CORNER_UP | TILE_ELEMENT_SLOPE_W_CORNER_UP,
-      TILE_ELEMENT_SLOPE_N_CORNER_UP | TILE_ELEMENT_SLOPE_E_CORNER_UP },
+    { SURFACE_STYLE_FLAG_RAISE_OR_LOWER_BASE_HEIGHT | kTileSlopeNCornerUp | kTileSlopeECornerUp,
+      SURFACE_STYLE_FLAG_RAISE_OR_LOWER_BASE_HEIGHT | kTileSlopeNCornerUp | kTileSlopeECornerUp,
+      SURFACE_STYLE_FLAG_RAISE_OR_LOWER_BASE_HEIGHT | kTileSlopeNCornerUp | kTileSlopeECornerUp,
+      SURFACE_STYLE_FLAG_RAISE_OR_LOWER_BASE_HEIGHT | kTileSlopeNCornerUp | kTileSlopeECornerUp,
+      0,
+      kTileSlopeNCornerUp,
+      kTileSlopeECornerUp,
+      kTileSlopeNCornerUp | kTileSlopeECornerUp,
+      0,
+      kTileSlopeNCornerUp,
+      kTileSlopeECornerUp,
+      kTileSlopeNCornerUp | kTileSlopeECornerUp,
+      0,
+      kTileSlopeNCornerUp,
+      kTileSlopeECornerUp,
+      kTileSlopeNCornerUp | kTileSlopeECornerUp,
+      SURFACE_STYLE_FLAG_RAISE_OR_LOWER_BASE_HEIGHT | kTileSlopeNCornerUp | kTileSlopeECornerUp,
```

Comment:
Maybe `SURFACE_STYLE_FLAG_RAISE_OR_LOWER_BASE_HEIGHT` is another good one to tackle in this PR?

Fwiw, in OpenLoco, we've called it `SurfaceSlope::requiresHeightAdjustment`.

Reply:
Good idea, pushed a commit to update it.